AppDB/groomer.py

```diff
@@ -980,7 +1377,11 @@ def main():
   logging.debug("Trying to get groomer lock.")
   if ds_groomer.get_groomer_lock():
     logging.info("Got the groomer lock.")
-    ds_groomer.run_groomer()
+    try:
```

Comment:
I just read that doing something like 
import os
os.nice(20)
allow to nice a program. Should the groomer be niced?